.zipignore

```diff
@@ -0,0 +1,14 @@
+*wpps-scripts/*
+*.git/*
+*.gitignore*
+*node_modules/*
+*package.json*
+*package-lock.json*
+*.circleci/*
+*composer.json*
+*composer.lock*
+*css/src/*
+*js/src/*
+*tests/*
+*postcss.config.js*
+*tailwind.config.js*
```

Comment:
This idea is taken from John's [work](https://github.com/studiopress/genesis-blocks/blob/08b006ed432b61075ea93ce24d5a697edf6be018/.zipignore) on Genesis Blocks